frontend/src/components/episode-live/episode-live.tsx

```diff
@@ -3,12 +3,13 @@ import {
   record as recordActions,
   episode as episodeActions,
```

Comment:
```suggestion
```
remove it
we can use only `recordActions` here

Reply:
> remove it
> we can use only `recordActions` here

functionality of episode loading, changing episode status and loading episode comments move to record action?